inc/saiipmcgroup.h

```diff
@@ -93,11 +93,24 @@ typedef enum _sai_ipmc_group_member_attr_t
      * @brief IPMC output id
      *
      * @type sai_object_id_t
-     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @flags CREATE_ONLY
      * @objects SAI_OBJECT_TYPE_ROUTER_INTERFACE, SAI_OBJECT_TYPE_TUNNEL
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
```

Comment:
Review Comment: Can we use SAI_OBJECT_TYPE_NEXT_HOP as an additional type on line 97?